hvplot/converter.py

```diff
@@ -2060,7 +2060,11 @@ def ohlc(self, x=None, y=None, data=None):
             o, h, l, c = y
         neg, pos = self.kwds.get('neg_color', 'red'), self.kwds.get('pos_color', 'green')
         color_exp = (dim(o)>dim(c)).categorize({True: neg, False: pos})
-        ds = Dataset(data, [x], [o, h, l, c] + self.hover_cols)
+        ohlc_cols = [o, h, l, c]
+        if x in self.hover_cols:
+            self.hover_cols.remove(x)
```

Comment:
Can this problem if `x` is `a` or `l` and `self.hover_cols` is all?

Reply:
Why would `x` be equal to `l`?

Reply:
Sorry. I just took all the letters of `all` and did not see that it was already part of the `ohlc` columns.